libraries/AP_Scripting/lua_scripts.cpp

```diff
@@ -393,7 +393,7 @@ void lua_scripts::run(void) {
             const uint32_t runEnd = AP_HAL::micros();
             const uint32_t endMem = lua_gc(L, LUA_GCCOUNT, 0) * 1024 + lua_gc(L, LUA_GCCOUNTB, 0);
             if (_debug_level > 1) {
-                gcs().send_text(MAV_SEVERITY_DEBUG, "Lua: Time: %d Mem: %d", runEnd - loadEnd, endMem - startMem);
+                gcs().send_text(MAV_SEVERITY_DEBUG, "Lua: Time: %u Mem: %u", (unsigned int)(runEnd - loadEnd), (unsigned int)(endMem - startMem));
```

Comment:
The memory one isn't actually correct, I am actually looking for a signed result from this expression, even though I'm giving it unsigned values. The correct fix is to just change endMem (and startMem) to both be `const int` instead of `const uint32_t` and yes, it is `int` not `int32_t`

Reply:
@WickedShell, ok, I think I've fixed it up as you've suggested